core/api/test/bats/helpers/setup-and-teardown.bash

```diff
@@ -68,21 +68,6 @@ add_callback() {
   exec_graphql "$token_name" 'callback-endpoint-add' "$variables"
 }
 
-remove_callbacks() {
```

Comment:
why is this not needed?

Reply:
arvin remove the use of callback in ln-receive and this is trigger some confusing message that took me a while to understand.

and I think that is fine not remove the callbacks in the callback test.